src/dmd/backend/backconfig.d

```diff
@@ -74,6 +75,23 @@ version (MARS)
     tytab[TYchar] |= TYFLuns;
     bool mscoff = model & 1;
     model &= 32 | 64;
+
+    if (dwarf != 3 && dwarf != 4)
```

Comment:
You allow `-gdwarf=5` on the command-line, yet issue an error here?

Reply:
Yes and no. I want to merge the first part of https://github.com/dlang/dmd/pull/11426 after that PR is merged.

Reply:
I allowed DWARFv5 anyway.